tests/integration/views/surveys/help/test_surveys_help_info_about_the_ons.py

```diff
@@ -176,7 +176,7 @@ def test_create_message_post_something_else_failure(self, mock_request, send_mes
 
         self.assertEqual(response.status_code, 200)
         self.assertIn("Error: ".encode(), response.data)
-        self.assertIn('<span class="ons-panel__assistive-text ons-u-vh">Error: </span>'.encode(), response.data)
+        self.assertIn('<span class="ons-panel__assistive-text ons-u-vh"'.encode(), response.data)
```

Comment:
I'm not sure whether to leave these in or not but they do check that the page adds a hidden label to the error for accessibility